<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPlaylist</class>
 <widget class="QWidget" name="SelectPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>301</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="defaultPlaylistBox">
     <property name="title">
      <string>Default Channel list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QRadioButton" name="presetPlaylistRadio">
        <property name="text">
         <string>Preset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="browseLayout">
        <item>
         <widget class="QLineEdit" name="playlistLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="browsePlaylistButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetPlaylistButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="customPlaylistRadio">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="presetsBox">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioSiol2">
        <property name="text">
         <string notr="true">SiOL - MPEG-2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioSiol4">
        <property name="text">
         <string notr="true">SiOL - MPEG-4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioT2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true">T-2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radioTus">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string notr="true">Tu≈° Telekom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSlovenia">
        <property name="text">
         <string>&lt;b&gt;Slovenia&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelGlobal">
        <property name="text">
         <string>&lt;b&gt;Global&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioWorldTv">
        <property name="text">
         <string notr="true">World TV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetsBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>playlistLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>browsePlaylistButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPlaylistRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>resetPlaylistButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
